### PR TITLE
Update Meeting-Summaries.txt

### PR DESCRIPTION
**Pull request by oriKatzir**
updated meeting 4

### CODE REVIEW

File: Meeting-Summaries.txt [hunk 1]
```diff
@@ -27,3 +27,12 @@ Meeting Summary 3: (without Liam)
 Goals for the next meeting:
 * Go through the tasks in Jira.
 * Go through chapters 3 and 4 (as much as possible).
+
+Meeting Summary 4: (with Liam and Ariel)
+* We informed Liam about the project requirements Areiel Rot dissmissed us from and that Liam is incharge with grwading our project.
```
Comment (bananabajanana) [inline Meeting-Summaries.txt]: Spelling mistake in the word "grading"

**Review by bananabajanana — CHANGES_REQUESTED** [review]
Please fix the problems :)

File: Meeting-Summaries.txt (hunk 1 above)
Comment (oriKatzir) [reply Meeting-Summaries.txt]: Done boss 👍

**Review by bananabajanana — COMMENTED** [review]
Good Job !

**Review by bananabajanana — APPROVED** [review]
good job!

**Pull request merged**